meshinfo.cc

```diff
@@ -168,23 +161,23 @@ int main(int argc, char *argv[]){
 
   }
 
+
   switch (dim){
 
     case 2:
-
         if (input_flag){
-          Triangulation<2> tria2 = reading_file<2>(input_file);
```

Comment:
This doesn't work as this creates a new local variable ``tria2`` that "shadows" the outer variable. It gets destroyed 2 lines below this and so ``writing_file`` never sees the loaded data.